prov/rxm/src/rxm.h

```diff
@@ -198,11 +198,14 @@ struct rxm_iov {
 
 enum rxm_buf_pool_type {
 	RXM_BUF_POOL_RX		= 0,
-	RXM_BUF_POOL_TX_MSG	= 1,
-	RXM_BUF_POOL_TX_TAGGED	= 2,
-	RXM_BUF_POOL_TX_ACK	= 3,
-	RXM_BUF_POOL_TX_LMT	= 4,
-	RXM_BUF_POOL_MAX_VAL	= 5,
+	RXM_BUF_POOL_START	= RXM_BUF_POOL_RX,
+	RXM_BUF_POOL_TX_MSG,
+	RXM_BUF_POOL_TX_START	= RXM_BUF_POOL_TX_MSG,
+	RXM_BUF_POOL_TX_TAGGED,
+	RXM_BUF_POOL_TX_ACK,
+	RXM_BUF_POOL_TX_LMT,
+	RXM_BUF_POOL_TX_END	= RXM_BUF_POOL_TX_LMT,
+	RXM_BUF_POOL_END,
```

Comment:
Merging this PR, but it's a little confusing to have TX_END be equal to the last value, but POOL_END be one more.  Renaming POOL_END to ... uhm ... POOL_COUNT or POOL_MAX would help some.

Reply:
I'll have some changes to this code soon, will fix this